source/material_model/visco_plastic.cc

```diff
@@ -916,6 +921,14 @@ namespace aspect
                              "is not automatically used. Values of 100e6--1000e6 $Pa$ have been used "
                              "in previous models. Units: $Pa$");
 
+          // Temperature in viscosity laws to include an adiabat (note units of K/Pa)
+          prm.declare_entry ("Adiabat temperature gradient for viscosity", "0.0", Patterns::Double(0),
+                             "Add an adiabatic temperature gradient to the temperature used in the flow law "
+                             "so that the activation volume is consistent with what one would use in a "
+                             "earth-like (compressible) model. Default is set so this is off. "
+                             "Using a pressure gradient of 32436 Pa/m, then a value of "
```

Comment:
Could you add a note here that this is a linear approximation of the real adiabatic gradient? In practice that will probably be good enough for the upper mantle, but for whole mantle convection models this would not be very helpful (there one would use the real adiabatic gradient instead).

Reply:
Yes, this is a good point and I have added a note to the description.